CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# dbt_servicenow v0.3.0
+[PR #10](https://github.com/fivetran/dbt_servicenow/pull/10) includes the following updates:
+
+## 🚨 Breaking Changes
+-  The following fields were removed from the `servicenow__task_enhanced` model as `problem_first_reported_by_task_value` was incorrectly assumed to have corresponding `sys_user` values, but in actuality is related to the task table and therefore have no user fields.
+  - `problem_reporter_email, problem_reporter_manager_value, problem_reporter_department_value, problem_reporter_name, problem_reporter_roles`
+
+## Feature Updates
```

Comment:
A new table was added to the staging models. We should be calling that out in the feature updates as well.

Reply:
added